processing/process_states.py

```diff
@@ -1,20 +1,24 @@
 class ProcessStates:
+    Uploaded = 1
     PassedQAQC = 2
-    IssuesFound = 4
-    FailedQAQC = 5
-    FailedRepair = 6
-    FailedRepairRetire = 16
-    AutoRepair = 14
-    AutoRepairRetire = 15
-    FinishedQAQC = 22
-    PublishedBASEBADM = 23
-    PassedCurator = 28
-    RepublishReport = 29
-    GeneratedBASE = 30
-    UpdatedBASEBADM = 31
-    BASEGenFailed = 32
-    BADMUpdateFailed = 33
-    BASEBADMPubFailed = 34
-    ArchiveUploaded = 36
-    RetiredForReprocessing = 38
-    InitiatedPreBASERegen = 39
+    IssuesFound = 3  # 4
+    FailedQAQC = 4  # 5
+    FailedRepair = 5  # 6
+    FailedRepairRetire = 11  # 16
+    AutoRepair = 9  # 14
+    AutoRepairRetire = 10  # 15
+    FinishedQAQC = 14  # 22
+    StartBASEGen = 12
+    FilesCombined = 13
+    PublishedBASEBADM = 15  # 23
+    CombinerFailed = 16
+    PassedCurator = 20  # 28
+    RepublishReport = 21  # 29
+    GeneratedBASE = 22  # 30
+    UpdatedBASEBADM = 23  # 31
+    BASEGenFailed = 24  # 32
+    BADMUpdateFailed = 25  # 33
+    BASEBADMPubFailed = 26  # 34
+    ArchiveUploaded = 28  # 36
+    RetiredForReprocessing = 30  # 38
+    InitiatedPreBASERegen = 31  # 39
```

Comment:
Not sure if this is in the plan, but can we pull the values from the actual table?

Reply:
Use names.

Reply:
done